py/build.gradle

```diff
@@ -3,6 +3,13 @@ apply from: "$rootDir/gradle/utils.gradle"
 ext {
     FS = File.separator
     FPS = File.pathSeparator
+
+    def command = ["python", "py/pythonVersion.py"]
```

Comment:
yup, that works, but I mean if similar approach in your case `commandLine getOsSpecificCommandLine(["python", "c"])` didn't work as well ( not considering the stdout at the moment )